outputs.tf

```diff
@@ -1,3 +1,16 @@
 output "id" {
   value = "${null_resource.default.triggers.id}"
-}
\ No newline at end of file
+}
+
```

Comment:
Can we `output` a tag map?

Then add: (or like this)
```
output "tags" {
   value = "${map("Name", null_resource.default.triggers.id, "Namespace", ...)}"
}

In the `tags` map, we will want `Name` to be `null_resource.default.triggers.id` because `Name` has a special meaning in the AWS console - it is what gets displayed on all resources. We want to use the disambiguated `id` field so that resources are easily differentiated from each other.

![image](https://user-images.githubusercontent.com/52489/28836998-158a3990-76a0-11e7-9dd2-d037a4221e6a.png)